pyzx/__init__.py

```diff
@@ -43,7 +45,9 @@
 from . import simulate
 from . import editor
 from . import routing
-from . import sparsify
+from . import scores
+from . import simulated_annealing
+from . import genetic
 from .routing.parity_maps import CNOT_tracker
```

Comment:
Could we maybe put all of these under a single module, that might be a little less messy.